### PR TITLE
Loss update

### PR DESCRIPTION
**Pull request by kprokofi**

### CODE REVIEW

**Comment by kprokofi** [comment]
WIP - seems like loss isn't working now properly

File: torchreid/losses/asl.py [hunk 1]
```diff
@@ -113,21 +108,16 @@ def forward(self, cos_theta, targets):
             balance_koeff_pos = (K - C) / K # balance loss
             balance_koeff_neg = 1 - balance_koeff_pos
         elif self.asymmetric_focus:
-            balance_koeff_pos = 1
-            balance_koeff_neg = 1
+            # Asymmetric Focusing
```
Comment (sovrasov) [inline torchreid/losses/asl.py]: This comment is useless, flag name is verbose enough

**Comment by kprokofi** [comment]
- [x] Fix ASL and AM_binary loss. Validated. Added reduction by batch and other changes
- [x] Remove Q2L

**Comment by kprokofi** [comment]
@sovrasov , ready for review

File: scripts/default_config.py [hunk 2]
```diff
@@ -618,6 +616,7 @@ def model_kwargs(cfg, num_classes):
         'type': cfg.model.type,
         'self_challenging_cfg': cfg.model.self_challenging_cfg,
         'hidden_dim': cfg.model.transformer.hidden_dim,
+        'similarity_adjustment': cfg.loss.am_binary.amb_t,
```
Comment (sovrasov) [inline scripts/default_config.py]: It looks like similarity_adjustment should be boolean

Comment (sovrasov) [reply scripts/default_config.py]: cfg.loss.am_binary.amb_t != 0 or smth as soon as you excluded cfg.loss.am_binary.sym_adjustment from config

**Review by sovrasov — APPROVED** [review]

**Pull request merged**